packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -37,28 +42,40 @@ export const NotificationBadge: React.FunctionComponent<NotificationBadgeProps>
   icon = <BellIcon />,
   className,
   isExpanded = false,
+  hasAnimation = false,
   ...props
 }: NotificationBadgeProps) => {
+  const [isAnimating, setIsAnimating] = useState(hasAnimation);
```

Comment:
We'll need to add back that useEffect to update this state with the hasAnimation prop in the dependency array. Trying something out locally, having some other state to pass into hasAnimation, the animation doesn't get triggered when that updates.

Reply:
+1

Reply:
yes, definitely... should be good now